<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>load_dialog</class>
 <widget class="QDialog" name="load_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#load_dialog{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 5, 68, 255), stop:1 rgba(10, 55, 255, 255));}</string>
  </property>
  <widget class="QListWidget" name="files_list">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>511</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial&quot;;color:rgb(255, 255, 255);
background-color:rgba(0,0,0,0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_field">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>511</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
